coin/coin_presentation/src/main/java/com/pavluyk/coin_presentation/viewmodel/CoinDetailedViewModel.kt

```diff
@@ -9,7 +9,7 @@ import com.pavluyk.coin_domain.usecases.FetchDetailedDataUseCase
 import kotlinx.coroutines.launch
 
 class CoinDetailedViewModel(
-    private val fetchDetailedDataUseCase: FetchDetailedDataUseCase,
+    private val fetchDetailedData: FetchDetailedDataUseCase,
```

Comment:
generally dont like naming like this, mb if in context of single-purpose app "fetch SOME ABSTRACT data" looks not so bad, but probably when you now what EXACT you fetch it looks better